<ui version="4.0" >
 <class>SvnUpdateOptionWidget</class>
 <widget class="QWidget" name="SvnUpdateOptionWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>387</width>
    <height>252</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="nonrecurseChk" >
     <property name="text" >
      <string>Update immediate children only (non-recursive)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QCheckBox" name="ignoreexternalChk" >
     <property name="text" >
      <string>Ignore external entries</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Select Revision</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="SvnRevisionWidget" native="1" name="revisionWidget" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>SvnRevisionWidget</class>
   <extends>QWidget</extends>
   <header>svnrevisionwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
  <include location="local" >knuminput.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
